File: 프로그래머스/0/120866. 안전지대/안전지대.java
class Solution {
    public int solution(int[][] board) {
        int answer = 0;
        int len = board.length;
        for(int i=0; i<len; i++){
            for(int j=0; j<len; j++){
                int cur = 1;
                if(
                    (board[i][j] == 1)
                    || (j<len-1 && board[i][j+1] == 1)
                    || (i<len-1 && j<len-1 && board[i+1][j+1] == 1)
                    || (i<len-1 && board[i+1][j] == 1)
                    || (i<len-1 && j>0 && board[i+1][j-1] == 1)
                    || (j>0 && board[i][j-1] == 1)
                    || (i>0 && j>0 && board[i-1][j-1] == 1)
                    || (i>0 && board[i-1][j] == 1)
                    || (i>0 && j<len-1 && board[i-1][j+1] == 1)
                ){
                    cur = 0;
                }
                answer += cur;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120866. 안전지대/안전지대.py
def solution(board):
    answer = 0
    length = len(board)
    for i in range(length):
        for j in range(length):
            cur = 1
            if (
                (board[i][j] == 1)
                or (j<length-1 and board[i][j+1] == 1)
                or (i<length-1 and j<length-1 and board[i+1][j+1] == 1)
                or (i<length-1 and board[i+1][j] == 1)
                or (i<length-1 and j>0 and board[i+1][j-1] == 1)
                or (j>0 and board[i][j-1] == 1)
                or (i>0 and j>0 and board[i-1][j-1] == 1)
                or (i>0 and board[i-1][j] == 1)
                or (i>0 and j<length-1 and board[i-1][j+1] == 1)
            ):
                cur = 0
            answer += cur
    return answer